static/app/views/replays/detail/errorList.tsx

```diff
@@ -0,0 +1,191 @@
+import {PanelTable} from 'sentry/components/panels';
+import Placeholder from 'sentry/components/placeholder';
+import {DEFAULT_STREAM_GROUP_STATS_PERIOD} from 'sentry/components/stream/group';
+import GroupChart from 'sentry/components/stream/groupChart';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {Group, NewQuery, PageFilters} from 'sentry/types';
+import DiscoverQuery from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import theme from 'sentry/utils/theme';
+import useApi from 'sentry/utils/useApi';
+import {useLocation} from 'sentry/utils/useLocation';
+import useMedia from 'sentry/utils/useMedia';
+import useOrganization from 'sentry/utils/useOrganization';
+import withPageFilters from 'sentry/utils/withPageFilters';
+
+type Props = {
+  projectId: string;
+  replayId: string;
+  selection: PageFilters;
+};
+const columns = [t('Issue'), t('Graph'), t('Events'), t('Users')];
+
+function ErrorList(props: Props) {
+  const organization = useOrganization();
+  const location = useLocation();
+  const api = useApi();
+  const isScreenLarge = useMedia(`(min-width: ${theme.breakpoints[2]})`);
+
+  const [issues, setIssues] = useState<Group[]>([]);
+  const [issueStats, setIssuesStats] = useState<Group[]>([]);
+
+  const getEventView = () => {
+    const {selection} = props;
+    const eventQueryParams: NewQuery = {
+      id: '',
+      name: '',
+      version: 2,
+      fields: ['count(issue)', 'issue'],
+      environment: selection.environments,
+      projects: selection.projects,
+      query: `replayId:${props.replayId} AND event.type:error`,
+    };
+    const result = EventView.fromNewQueryWithLocation(eventQueryParams, location);
+    return result;
+  };
+
+  useEffect(() => {
+    const options = {
+      includeAllArgs: true,
+      query: {
+        collapse: 'stats',
+        expand: ['owner', 'inbox'],
+        project: props.projectId,
+        query: `replayId:${props.replayId}`,
+      },
+    };
+
+    api
+      .requestPromise(`/organizations/${organization.slug}/issues/`, options)
+      .then(async result => {
+        setIssues(result[0]);
+        const statsOptions = {
+          includeAllArgs: true,
+          query: {
+            project: props.projectId,
+            groups: result[0].map(issue => issue.id),
+            query: `replayId:${props.replayId}`,
+          },
+        };
+        const issuesResults = await api.requestPromise(
+          `/organizations/${organization.slug}/issues-stats/`,
+          statsOptions
+        );
+        setIssuesStats(issuesResults[0]);
+      });
+  }, [api, organization.slug, props.replayId, props.projectId]);
+
+  const renderTableRow = error => {
+    const matchedIssue = issues.find(
+      issue => parseInt(issue.id, 10) === error['issue.id']
+    );
+    const matchedIssueStats = issueStats.find(
+      issueStat => parseInt(issueStat.id, 10) === error['issue.id']
+    );
```

Comment:
When we fetch the data it might be more efficient to store `issues` and `issueStats` indexed by the `id`.

```
import keyBy from 'lodash/keyBy';

// Init state
const [issuesById, setIssuesById] = useState<Record<string, Group>>({});

// Fetch and store
const issues = [{id: 1}, {id: 2}, ...];
setIssuesById(keyBy(issues, 'id'));

// Find match
const matchedIssue = issuesById[error['issue.id']];
```

Reply:
TIL keyBy, that's handy